src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -85,9 +91,21 @@ class CodeLine(val repo: Repository,
     /**
      * Email address of the line's author.
      */
-    val email : String
+    val authorEmail : String
```

Comment:
I think we need to start thinking beyond email. It should be some sort of an email-derived id here. e.g. sha256(email). Eventually, we won't have emails in our database, so let's start preparing for it.

Reply:
Not sure what changes are required? It rather looks like a separate task with me, possibly involving changes in other hashers.

Reply:
per our chat: this patch doesn't send anything on the server yet, while hashing should happen before data is sent to the server, thus no changes are required for this PR.